### ISSUE TITLE
Internationalization: Use localized strings

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#742]
**User Story**

As a user, I want Platform.Bible to have the equipment in place to be able to display the UI in my language so I can work effectively.

**Description**

Depends on #741

Use the localization service and `useLocalizedStrings` hook to actually make the UI internationalized so it uses the English localized strings file (by default).

If #740 is finished, put localized strings in the associated extension where relevant.

**Comment by katherinejensen00** [comment]
Note: created https://github.com/paranext/paranext-core/issues/1094 to handle localizing Bible book names.

**Comment by tombogle** [comment]
I'm not sure if this is considered part of this issue, but I just tried switching to Spanish (which I just added as a locale) and I observed that I had to restart Platform.Bible to get the change to take effect.

**Comment by tjcouch-sil** [comment]
> I'm not sure if this is considered part of this issue, but I just tried switching to Spanish (which I just added as a locale) and I observed that I had to restart Platform.Bible to get the change to take effect.

That's intended functionality. We should probably add some kind of notification that you need to restart, but we don't have anything like that right now. I'd say that's not part of this issue.

**Comment by tombogle** [comment]
You mean, it's intended that you should have to restart? Is that a technical limitation with Electron, React and/or our approach to localization? If there were a way to get the UI to re-render using the new locale, that would be a good bit nicer for users.

**Comment by tjcouch-sil** [comment]
> You mean, it's intended that you should have to restart? Is that a technical limitation with Electron, React and/or our approach to localization? If there were a way to get the UI to re-render using the new locale, that would be a good bit nicer for users.

That was intended by design - I suppose to make devs' jobs easier. Otherwise, every time anyone gets a localized string, they would have to subscribe to updates and make sure to figure out some way to change the strings they used. Honestly, it's pretty easy to do this when we get strings on the frontend (`useLocalizedStrings` has it built in). It's not as convenient on the backend. 

I imagine that, if *none* of the UI currently updates when you change language, it's probably because the localization service isn't listening for updates to the language setting. If it were listening, you'd probably see everything that uses `useLocalizedStrings` (probably most of the UI) update and nothing that doesn't. I don't think we're manually subscribing to string updates anywhere (like anywhere on the backend that we use strings).

Comment on the design doc about this: https://docs.google.com/document/d/1dFSTR6bgt7n0HaXOvnV3Cc-z1R2S9Gb5KlNlIS4liQI/edit?disco=AAABEmHIqu8

**Comment by katherinejensen00** [comment]
Added localizations paranext-core, paratext-bible-extensions, and paratext-bible-internal-extensions. I also checked platform-bible-sample-extensions, but the strings in the verse ref view were already localized.
